.buildkite/pipeline.build.yml

```diff
@@ -1,3 +1,5 @@
+#ci:group=Rebuild Ray tests
```

Comment:
this is also not accurate.. I can update the readme file in another PR. change it into `pipeline.build*.yml`. 

this is more for tests (or anything) that does not need to run on the container that has ray already built and installed in the environment.